webserver/templates/user/scraper.js

```diff
@@ -170,7 +178,9 @@ function reportPage(response) {
 }
 
 function reportPageAndGetNext(response) {
-  document.getElementById("listen-progress-container").innerHTML = "<img src='{{ url_for('static', filename='img/listenbrainz-logo.svg', _external=True) }}' height='75'><br><br><i class='fa fa-cog fa-spin'></i> Sending page " + (numCompleted+1) + " of " + numberOfPages + " to ListenBrainz<br><span style='font-size:8pt'>Please don't navigate while this is running</span><br>";
+    if (page == 1) {
+      updateMessage("<i class='fa fa-cog fa-spin'></i> working<br><span style='font-size:8pt'>Please don't navigate while this is running</span>");
+    }
     reportPage(response);
```

Comment:
Yep! good catch:
https://github.com/metabrainz/listenbrainz-server/pull/31

I'll merge that pr into here